app/src/main/java/org/thoughtcrime/securesms/preferences/ClearAllDataDialog.kt

```diff
@@ -86,8 +99,12 @@ class ClearAllDataDialog : DialogFragment() {
                     binding.dialogDescriptionText.setText(R.string.dialog_clear_all_data_clear_device_and_network_confirmation)
                 }
                 Steps.DELETING -> { /* do nothing intentionally */ }
+                Steps.RETRY_LOCAL_DELETE_ONLY_PROMPT -> {
+                    binding.dialogDescriptionText.setText(R.string.clearDataErrorDescriptionGeneric)
+                    binding.clearAllDataButton.text = getString(R.string.clearDevice)
+                }
             }
-            binding.recyclerView.isGone = step == Steps.NETWORK_PROMPT
+            binding.recyclerView.isGone = step == Steps.NETWORK_PROMPT || step == Steps.RETRY_LOCAL_DELETE_ONLY_PROMPT
```

Comment:
this was probably only `isGone` because there was only one step to check, now there's 2, it's simpler to say `isVisible = either of the other two`, which is kinda better anyway, because we're checking the 2 steps that have items, not that it's not in a step that doesn't have items... yeah. Better still would be to extract the logic out of the view, but could be overkill here.

Reply:
Flipped logic to use `isVisible` rather than `isGone`